bip-0039/portuguese.txt

```diff
@@ -0,0 +1,2048 @@
+abacaxi
+abaixo
+abalo
+abelha
+abismo
+absinto
+absurdo
+abutre
+acaso
+aceito
+achar
+acerola
+acidez
+acima
+acne
+acordo
+adega
+adesivo
+adiante
+adiposo
+adjunto
+adorar
+adubo
+adverso
+afago
+aferir
+aflito
+afoito
+afta
+agito
+agonia
+agora
+agrado
+agreste
+agulha
+alameda
+alarme
+aldeia
+alecrim
+alegria
+alelo
+alface
+algema
+alheio
+alho
+alicate
+alpe
+alpiste
+aluguel
+aluno
+alvo
+amador
+amarelo
+ambiente
+ameixa
+amido
+amaro
+ampola
+amostra
+ampola
+anarquia
+anchova
+anel
+anedota
+anexo
+angular
+anjo
+anotado
+anta
+antigo
+anzol
+apache
+apatia
+apego
+apenas
+apesar
+apetite
+apito
+aplauso
+apoio
+aposta
+apto
+arame
+aranha
+arapuca
+arara
+areia
+argila
+argola
+arma
+arquivo
+arraial
+arruela
+arsenal
+arvoredo
+aspargo
+aspecto
```

Comment:
Can also be written as `aspeto`. Maybe replace? See https://dicionario.priberam.org/aspecto.

Reply:
We have to replace this one, for sure. I didn't realize you guys use `aspeto`. Suggestions?
I'll reply the other ones shortly.

Reply:
aspersor as alternative?